---
title: ''
layout: archive
permalink: /publications/
author_profile: yes
---

{% include base_path %}

For a full list of publications, please visit my **[Google Scholar profile](https://scholar.google.de/citations?user=c5EwcVAAAAAJ)**.

List of publications:

2024
*Giménez, A., Del Giudice, M., López, P., Guaimas, F., **Sámano-Sánchez, H.**, Gibson, T., Chemes, L., Arregui, C., Ugalde, J., Czibener, C. Brucella NpeA is a secreted Type IV effector containing an N-WASP binding short linear motif that promotes niche formation. *mBio*, Online ahead of print, 2024. doi: [10.1128/mbio.00726-24](https://journals.asm.org/doi/10.1128/mbio.00726-24).

2023
* **Sámano-Sánchez, H.**, Gibson, T.J., Chemes, L.B. Using Linear Motif Database Resources to Identify SH2 Domain Binders. *Methods Mol Biol*, 2023. doi: [10.1007/978-1-0716-3393-9_9](https://doi.org/10.1007/978-1-0716-3393-9_9).

2022
* Kumar, K., ..., **Sámano-Sánchez, H.**, ..., Gibson, T.J. The eukaryotic linear motif resource: 2022 release. *Nucleic Acids Research*, 2022. doi: [10.1093/nar/gkab975](https://doi.org/10.1093/nar/gkab975).

2021
* Meszaros, B., **Sámano-Sánchez, H.**, ..., Chemes, L.B., Gibson, J.T. Short linear motif candidates in the cell entry system used by SARS-CoV-2 and their potential therapeutic implications. *Science Signaling*, 2021. doi: [10.1126/scisignal.abd0334](https://doi.org/10.1126/scisignal.abd0334).

2020
* **Sámano-Sánchez, H.**, Gibson, J.T. Mimicry of Short Linear Motifs by Bacterial Pathogens: A Drugging Opportunity. *Trends in Biochemical Sciences*, 2020. doi: [10.1016/j.tibs.2020.03.003](https://doi.org/10.1016/j.tibs.2020.03.003).

2019
* Kumar, M., Gouw, M., **Sámano-Sánchez, H.**, ..., Gibson, J.T. ELM—the eukaryotic linear motif resource in 2020. *Nucleic acids research*, 2019. doi: [10.1093/nar/gkz1030](https://doi.org/10.1093/nar/gkz1030).

2018
* Sampietro, D., **Sámano-Sánchez, H.**, et al. Conserved SQ and QS motifs in bacterial effectors suggest pathogen interplay with the ATM kinase family during infection. *bioRxiv*, 2018. doi: [10.1101/364117](https://www.biorxiv.org/content/early/2018/07/09/364117.abstract).

2017
* Gouw M., Michael, S., **Sámano-Sánchez, H.**, ..., Gibson, J.T. The eukaryotic linear motif resource–2018 update. *Nucleic acids research*, 2017. doi: [10.1093/nar/gkx1077](https://doi.org/10.1093/nar/gkx1077).

* Kosaisavee, K., ..., **Sámano-Sánchez, H.**, ..., Rénia, L., Russell, B. Strict tropism for CD71+/CD234+ human reticulocytes limits the zoonotic potential of *Plasmodium cynomolgi*. *Blood*, 2017. doi: [10.1182/blood-2017-02-764787](https://doi.org/10.1182/blood-2017-02-764787).

* Gouw, M., **Sámano-Sánchez, H.**, ..., Dinkel, H. Exploring Short Linear Motifs Using the ELM Database and Tools. *Current Protocols in Bioinformatics*, 2017. doi: [10.1002/cpbi.26](https://doi.org/10.1002/cpbi.26).

2016
* Godoy-Lozano, E.E., ..., **Sámano-Sánchez, H.**, ..., Martínez-Barnetche, J. Lower IgG somatic hypermutation rates during acute dengue virus infection is compatible with a germinal center-independent B cell response. *Genome Medicine*, 2016. doi: [10.1186/s13073-016-0276-1](https://doi.org/10.1186/s13073-016-0276-1).

2015
* Cortina-Ceballos, B., ..., **Sámano-Sánchez, H.**, ..., Martínez-Barnetche, J. Longitudinal analysis of the peripheral B cell repertoire reveals unique effects of immunization with a new Influenza virus strain. *Genome Medicine*, 2015. doi: [10.1186/s13073-015-0239-y](https://doi.org/10.1186/s13073-015-0239-y).

* Cortina-Ceballos, B.**<sup>\*</sup>**, Godoy-Lozano, E.E.**<sup>\*</sup>**, **Sámano-Sánchez, H.<sup>\*</sup>**, ..., Martínez-Barnetche, J. Reconstructing and mining the B cell repertoire with ImmunediveRsity. *mAbs*, 2015. doi: [10.1080/19420862.2015.1026502](https://doi.org/10.1080/19420862.2015.1026502).

2011
* Rosenblueth, M., Sayavedra, L., **Sámano-Sánchez, H.**, Roth, A., Martínez-Romero, E. Evolutionary Relationships of Flavobacteria and Enterobacterial Endosymbionts with their Scale Insect Hosts (Hemiptera: Coccoidea). *Journal of Evolutionary Biology*, 2011. doi: [10.1111/j.1420-9101.2012.02611.x](https://doi.org/10.1111/j.1420-9101.2012.02611.x).

**<sup>\*</sup>** Denotes co-first authors
